App/main.js

```diff
@@ -96,6 +97,25 @@ const editor = Editor(ui)
 ui.levelText.setAttribute('hide', true)
 ui.veil.setAttribute('hide', true)
 
+// Detect mobile devices and show the warning
+const ua = navigator.userAgent || navigator.vendor || window.opera
+let mobile = false
+
+if (
+  /(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(
```

Comment:
Nice, I might use this instead of https://github.com/hackclub/sinerider/pull/320/commits/cfaa3cd94ca91179711d7c5637dbd16eff2a28b6